CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Upgrade to version 6.31.0 of the upstream chart with grafana 9.0.1.
```

Comment:
I prefer when it says "**from** xxx to xxx", especially when there's a gap, it helps reading the upstream changelog for breaking changes.

Reply:
:ack: